docker/app/Dockerfile

```diff
@@ -2,21 +2,19 @@
 ARG ENVIRONMENT=${ENVIRONMENT:-'production'}
```

Comment:
I believe this can stay and still be able to skip build steps with `--target=foo` if using BuildKit. But I'll do some more extensive testing to prove or disprove that, and open a new PR if we do have to get rid of the pseudo-conditionals completely.

Reply:
Just proved it: watching the `docker compose` build logs carefully, there's no `webpack:dev` step run when the target is production, and no `webpack:prd` step run when the target is development. BuildKit's skipping of extra steps is enough to make the "fake conditional" work properly, not running the unneeded build steps, as long as the correct target is selected.